src/storage/hummock_sdk/src/change_log.rs

```diff
@@ -148,12 +203,12 @@ impl<T> TableChangeLogCommon<T> {
 
     pub fn truncate(&mut self, truncate_epoch: u64) {
         while let Some(change_log) = self.0.front()
-            && *change_log.epochs.last().expect("non-empty") <= truncate_epoch
+            && *change_log.epochs.last().expect("non-empty") < truncate_epoch
```

Comment:
Is this a bug fix?

Reply:
Not really. It just delays the truncation of the log on `truncate_epoch`, and is not wrong before this PR. After this PR, we need to keep the `truncate_epoch` in the change log.